# patches/zx+7.0.1.patch
# A patch that modifies the appearance of the spinner.

diff --git a/node_modules/zx/build/experimental.js b/node_modules/zx/build/experimental.js
index e7dd0bd..8576d09 100644
--- a/node_modules/zx/build/experimental.js
+++ b/node_modules/zx/build/experimental.js
@@ -76,7 +76,7 @@ export async function spinner(title, callback) {
         title = '';
     }
     let i = 0;
-    const spin = () => process.stderr.write(`  ${'⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏'[i++ % 10]} ${title}\r`);
+    const spin = () => process.stderr.write(`${'⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏'[i++ % 10]}  ${title}\r`);
     return within(async () => {
         $.verbose = false;
         const id = setInterval(spin, 100);
@@ -86,6 +86,7 @@ export async function spinner(title, callback) {
         }
         finally {
             clearInterval(id);
+            process.stderr.write(`${title.split('').map(() => ' ').join('')}\r`)
         }
         return result;
     });
